src/api.js

```diff
@@ -8,6 +8,7 @@ let api = new API({
     'listens for tasks with associated notifications and handles requests to',
     'send emails and post pulse messages.',
   ].join('\n'),
+  name:         'notify',
```

Comment:
I'd prefer if this was just a single space rather than aligned with subsequent values.